content/applications/hr/lunch.rst

```diff
@@ -0,0 +1,313 @@
+the drop-down menu.
+
+If desired, add a photo for the category. Hover over the image box in the top-right, and click the
+:guilabel:`✏️ (pencil)` edit icon that appears. A file explorer pop-up window appears. Navigate to
+the image, then click :guilabel:`Open`.
+
+.. image:: lunch/category.png
+   :align: center
+   :alt: The category form, with the fields filled out for a Soup category.
+
+Alerts
+======
+
+It is possible to set up alerts that can either be displayed in the *Lunch* app, or be sent to
+specific employees via the *Discuss* app.
+
+No alerts are pre-configured by default. To set up an alert, navigate to :menuselection:`Lunch app
+--> Configuration: Alerts`. Click the :guilabel:`New` button in the top-left corner and a blank
+lunch alert form loads. Enter the following information on the form:
+
+- :guilabel:`Alert Name`: enter a name for the alert. This should be short and descriptive, such as
+  `New Lunch Vendor` or `Order by 11`. This field is **required**.
+- :guilabel:`Display`: select whether the alert is visible in the *Lunch* app (:guilabel:`Alert in
+  app)` or sent to employees via the *Discuss* app in a chat window (:guilabel:`Chat notification`).
+- :guilabel:`Recipients`: this field only appears if :guilabel:`Chat notification` is selected for
+  the :guilabel:`Display` option. Select who receives the chat alert. The options are:
+  :guilabel:`Everyone`, :guilabel:`Employee who ordered last week`, :guilabel:`Employee who ordered
+  last month`, or :guilabel:`Employee who ordered last year`.
+- :guilabel:`Location`: select the locations the alert should appear for from the drop-down menu.
+  Multiple locations can be selected. This field is **required**, therefore, if the alert applies
+  to all locations, select all the locations from the drop-down menu.
+- :guilabel:`Show Until`: if the alert should expire on a specific date, select the date from the
+  calendar picker.
+- :guilabel:`Active`: this option is on (appears green) by default. To turn off the alert, click the
+  toggle so that it no longer appears green.
+- :guilabel:`Message`: Enter the alert message in this field. This field is **required**.
+- :guilabel:`Notification Time`: select the days of the week the alert should be sent. By default,
+  all seven days are active. Click on a checkbox to change the setting from active to inactive.
+
+  If :guilabel:`Chat notification` was selected for the :guilabel:`Display` option, a
+  :guilabel:`Time` field also appears. Enter the time the chat message should be sent. Next, select
+  if the time is either :guilabel:`AM` or :guilabel:`PM` using the drop-down menu to the right of
+  the :guilabel:`Time` field.
+
+.. image:: lunch/alert.png
+   :align: center
+   :alt: An alert form with al the information filled out for a chat alert sent at 10:30 AM, asking
+         employees to submit orders by 11:30 AM.
+
+.. toctree::
+   :titlesonly:
```

Comment:
No need for a TOCTREE since there are no sub-docs
```suggestion
```

Reply:
I put that in there as a placeholder because I am adding the other docs now, lol. But I'll remove for now!